src/views/product-landing/components/product-landing-blocks/index.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/no-array-index-key */
+import { ProductLandingBlocksProps, ProductLandingBlock } from './types'
+import { HeadingBlock, LinkedCards, TutorialCards } from './blocks'
+
+function ProductLandingBlocks({ blocks }: ProductLandingBlocksProps) {
+  return (
+    <>
+      {blocks.map((block: ProductLandingBlock, idx: number) => {
```

Comment:
TLDR; is this component maps over the processed block props and renders the appropriate components.

Note that I still need to add `collection_cards` here - will handle that in an upcoming PR which will also fetch proper data for tutorials & collections (rather than just slugs as we have currently).